.github/workflows/fedora-rawhide-test.yml

```diff
@@ -9,4 +9,6 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Run Fedora Rawhide Test
-      run: sudo -E make -C scripts/ci fedora-rawhide CONTAINER_RUNTIME=podman BUILD_OPTIONS="--security-opt seccomp=unconfined"
+      # Make sure that XDG_RUNTIME_DIR environment variable is not set.
+      # https://github.com/checkpoint-restore/criu/issues/1942
+      run: sudo -E XDG_RUNTIME_DIR= make -C scripts/ci fedora-rawhide CONTAINER_RUNTIME=podman BUILD_OPTIONS="--security-opt seccomp=unconfined"
```

Comment:
why do we set -E in this case? Do we pass any env var-s?

Reply:
We need environment variables from the CI environment to distinguish between CI environments.